flatpak-flutter.yml

```diff
@@ -41,16 +42,20 @@ modules:
             BUNDLE_PATH: build/linux/arm64/release/bundle
       build-args:
         - --share=network
-      append-path: /usr/lib/sdk/llvm19/bin:/usr/lib/sdk/golang/bin
+      append-path: /usr/lib/sdk/llvm19/bin:/usr/lib/sdk/golang/bin:/run/build/Gopeed/flutter/bin
       prepend-ld-library-path: /usr/lib/sdk/llvm19/lib
       env:
         PUB_CACHE: /run/build/Gopeed/.pub-cache
     build-commands:
       - go build -mod=vendor -tags nosqlite -ldflags="-w -s -X github.com/GopeedLab/gopeed/pkg/base.Version=$(grep '^version:' ui/flutter/pubspec.yaml | cut -d ' ' -f 2 | cut -d '+' -f 1)"
-        -buildmode=c-shared -o ui/flutter/linux/bundle/lib/libgopeed.so github.com/GopeedLab/gopeed/bind/desktop
+        -buildmode=c-shared -o ui/flutter/linux/bundle/lib/libgopeed.so github.com/GopeedLab/gopeed/cmd/host
```

Comment:
When combining the two manifests, I noticed one is using `bind/desktop` and one `cmd/host`. I picked the one from the offline/publishing manifest, but this might need correction.

Reply:
well, I made some mistakes. I will make corrections after merged.